cypress/support/step-definitions/common-steps.js

```diff
@@ -530,3 +531,7 @@ When('I press keyboard {string} key times {int}', (key, times) => {
 When('I click on outside dialog in iFrame', () => {
   cy.get('#story-root').click({ force: true });
 });
+
+Then('totalRecords is set to {string} {word}', (totalRecords, element) => {
```

Comment:
Do we have a reason why this step in common ?

Reply:
yes, because we use it in three different components - pager, tableClassic and TableWithInputs